### DISCUSSION TITLE
how to solve ValueError: Message onnx.ModelProto exceeds maximum protobuf size of 2GB: 2868363175

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmdeploy#1465] [General]
ValueError: Message onnx.ModelProto exceeds maximum protobuf size of 2GB: 2868363175
- mmdeploy - ERROR - `mmdeploy.backend.tensorrt.onnx2tensorrt.onnx2tensorrt` with Call id: 1 failed. exit.

**Comment by AllentDan** [comment]
Hi, [ONNX](https://github.com/onnx/onnx/blob/fddb2b6d4ea3fb3dba751d884865042503260899/onnx/checker.py#L29) only supports models with the size of less than 2GB.
